src/backend/docker-compose.yml

```diff
@@ -118,5 +118,4 @@ volumes:
 
 # Define the absolute path for PROJECT_ROOT
 x-environment:
-  PROJECT_ROOT: ${PROJECT_ROOT}
```

Comment:
Remove this unnecessary change, in future make sure, all these unnecessary changes are not present